src/database.php

```diff
@@ -10,24 +10,28 @@ function getDbConn() {
   if (isset($test_db_connection)) {
     return $test_db_connection;
   } else {
-    return DriverManager::getConnection([
-      'url' => $_SERVER["DATABASE_URL"],
```

Comment:
Does this mean that specifying `DATABASE_URL` (as Heroku does) will no longer have any effect?